framework/bundles/org.jboss.tools.rsp.server/src/main/java/org/jboss/tools/rsp/server/model/ServerManagementModel.java

```diff
@@ -156,4 +166,8 @@ private IFileWatcherService createFileWatcherService() {
 		return new FileWatcherService();
 	}
 
+	private IJobManager createJobManager() {
```

Comment:
why have this extracted to it's private method? If protected I'd understand, subclasses could override it. Other than this, what's the benefit?